public/style.css

```diff
@@ -130,3 +129,176 @@ img {
   width: 100%;
   background-color: aqua;
 }
+
+/* below are for component productDetail */
+
+.productDetailContainer {
```

Comment:
Let's also alphabetize these class names.

I can foresee an instance down the road when we want to update some css on the page, and it would be a lot easier to find them if we tidy it up by sorting alphabetically.

Reply:
ok